pkg/hotplug-disk/hotplug-disk.go

```diff
@@ -76,6 +79,26 @@ func (h *hotplugDiskManager) GetHotplugTargetPodPathOnHost(virtlauncherPodUID ty
 	return "", fmt.Errorf("Unable to locate target path: %s", podpath)
 }
 
+// GetFileSystemDirectoryTargetPathFromHostView gets the directory path in the target pod (virt-launcher) on the host.
+func (h *hotplugDiskManager) GetFileSystemDirectoryTargetPathFromHostView(virtlauncherPodUID types.UID, volumeName string, create bool) (string, error) {
+	targetPath, err := h.GetHotplugTargetPodPathOnHost(virtlauncherPodUID)
+	if err != nil {
+		return "", err
+	}
+	directoryPath := filepath.Join(targetPath, volumeName)
+	exists, err := diskutils.FileExists(directoryPath)
+	if err != nil {
+		return "", err
+	}
+	if !exists && create {
+		err = os.Mkdir(directoryPath, 0750)
```

Comment:
if `err := os.Mkdir(directoryPath, 0750); err != nil...` with local err is the preferable style for such checks.